### ISSUE TITLE
Listing cheatsheets with tags not working on Windows

### ISSUE THREAD

**Issue by boydfields** [cheat/cheat#759]
**A description of the problem**
Using `cheat -l -t networking` or any other tag does not work on Windows. 
No results returned.

**cheat version info**
4.4.2

**Comment by jbrown123** [comment]
Agreed. I have the same issue on Windows 10. In addition, no tags (other than "community") are listed when asking for all tags using `cheat -T`

After playing around with it I've discovered that it's because of the line ending characters. Files with UNIX line ending (only a carriage return) do NOT display tags (e.g. all the community files). However, files with the WINDOWS line ending (CR + LF) DO correctly display tags.

For example, before changing line endings I get the following:
```
> cheat -v
4.4.2

> cheat -T
community
```

However, after running the [unix2dos](https://linux.die.net/man/1/unix2dos) utility on all the files in the ~\cheat\cheatsheets\community directory I get the following:

```
> cheat -T
apache
audio
bosh
build
checksum
community
compression
container
cryptography
database
date
deps
development
docker
ed
environment
environments
git
github
go
golang
hash
httpd
iac
jar
java
job
kitty
mac
networking
orchestration
packaging
periodic
perl
pulseaudio
python
reporting
ruby
salt
saltstack
scheduler
security
ssh
system reporting
time
vcs
verification
vim
virtualization
```

And now the `cheat -l -t <something>` commands work as expected.

**Comment by chrisallenlane** [comment]
Hi, all. Sorry for the delay getting to this. This should have been fixed at commit 8eafa5a (last night). Please upgrade to the newest release.
